stable/artifactory-ha/README.md

```diff
@@ -31,8 +31,10 @@ helm repo add jfrog https://charts.jfrog.io
 
 ### Install Chart
 To install the chart with the release name `artifactory-ha`:
+
+**NOTE:** Passing masterKey is mandatory for fresh install of chart (7.x Appversion)
 ```bash
-helm install --name artifactory-ha --set postgresql.postgresqlPassword=<postgres_password> jfrog/artifactory-ha
+helm install --name artifactory-ha --set postgresql.postgresqlPassword=<postgres_password> --set artifactory.masterKey=${MASTER_KEY} jfrog/artifactory-ha
```

Comment:
1. The `--set postgresql.postgresqlPassword=<postgres_password>` is not mandatory. Please remove.
2. Add the command for creating the MASTER_KEY before the `helm install` command.